QATCH/QModel/src/models/static_v4_fusion/v4_fusion.py

```
@@ -893,92 +907,197 @@ def predict(self,
         poi_map = {1: 'POI1', 2: 'POI2', 4: 'POI4', 5: 'POI5', 6: 'POI6'}
-        for poi_name, predictor in self.reg_predictors.items():
-            Log.d(self.TAG, f"Executing {poi_name} regression head.")
-            reg_result = predictor.predict(
-                df=df,
-                apply_smoothing=True
-            )
-            all_regression_results[poi_name] = reg_result
+
+        # Only process regression for POIs that were actually detected
+        all_regression_results = {}
+        detected_poi_names = {
```

Ruff (0.13.1)

Local variable `detected_poi_names` is assigned to but never used

Remove assignment to unused variable `detected_poi_names`

(F841)